doc/contributing/docs/markup/code.rst

```diff
@@ -41,15 +86,52 @@ Both options produce the same output:
 
 ..  include:: _includes/inline-code.rst
 
+
+Notes on using inline-code
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*   If you have expressions such as ``id==4``, you should format the whole
+    expression as code inline. Also,you can use the words 'equals',
+    'doesn't equal' or other similar words without formatting expression
```

Comment:
apostrophe (as single quote) in the phrase → use double quotes to avoid confusion

```suggestion
    expression as code inline. Also, you can use "equals",
    "doesn't equal" or other similar phrases without formatting expression
```